common/src/main/resources/assets/gtceu/compass/pages/en_us/circuits/circuit_board.xml

```diff
@@ -1,26 +1,45 @@
 <page>
- <h1>
-  Circuit Board
- </h1>
- <h3>
-  Source
- </h3>
- <text><style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br/>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/508290/">morningglory</style>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/2/">重生是希望</style><br/></style></text>
- <h2 font-color="0xff0000">
-  WARNING!!!
- </h2>
- <text isCenter="true">
-  <style color="0xff0000">The warning means this is not a reliable page.</style>
-  <br/>
-  <style bold="true" color="0xffaa00">Please do not report any mistakes on the page, either discord or github.</style>
-  <br/>
-  This page is translated from the Chinese wiki which hasn't been reviewed and polished yet. It may contain errors and inaccuracies. We are looking forward to your help by improving this page.
- </text>
- <text>
-  <br/>
-  <style>
-   Circuit substrates are the basic raw materials for synthesizing various circuits. As the stage progresses, you need to use more and more advanced circuit substrates.
-  </style>
-  <br/>
- </text>
+  <h1>Circuit Board</h1>
+  <h3>Source</h3>
+  <text>
+    <style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br></br>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/508290/">morningglory</style>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/2/">重生是希望</style><br></br>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://github.com/blackdragon2447">BlackDragon2447</style><br></br></style>
```

Comment:
In the case that you almost rewrite, you don't need to keep on mcmod ref if you want